LiskHQ/lisk-hub
# Implement the new design of update modal - Closes #2143

### What issue have I solved?

#2143

### How have I implemented/fixed it?

Implemented the design from https://projects.invisionapp.com/d/main#/console/17570736/369141341/preview but not inside the react app, as requested in #2143

### How has this been tested?

- Decrease version number to `1.1.0` in https://github.com/LiskHQ/lisk-hub/blob/178789e086eaae98e1251dab212131ec25c9d4f0/app/package.json#L3
- Run `npm run pack`
- Open Hub with `open dist/lisk-hub-mac-1.1.0.dmg`
- See the new update window
<img width="762" alt="Screenshot 2019-06-18 at 12 41 26" src="https://user-images.githubusercontent.com/1254342/59675852-82788780-91c6-11e9-8bb5-5de6828f185e.png">




### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
